R/list_distributions.R

```diff
@@ -41,26 +43,30 @@ list_distributions <- function(
     package = "epiparameter",
     mustWork = TRUE
   ))
-  
+
   # order params by pathogen, delay dist and study
   params <- params[order(
-    tolower(params$pathogen_id), 
-    tolower(params$type_id), 
-    tolower(params$study_id), 
+    tolower(params$pathogen_id),
+    tolower(params$type_id),
+    tolower(params$study_id),
     method = "radix"
   ), ]
-  
-  # filter by delay distribution
-  params <- params[params$type_id == delay_dist, ]
 
-  if (parameters == TRUE) {
-    output <- params
+  if (delay_dist != "all") {
+    # filter by delay distribution
+    params <- params[params$type_id == delay_dist, ]
   }
-  if (parameters == FALSE) {
+
+  if (isFALSE(parameters)) {
```

Comment:
Any reason to prefer `isFALSE(parameters)` to `!parameters` in this specific case?

If you want to keep `isFALSE()`, you'll need to add a dependency on R 3.5.0 in `DESCRIPTION`

Reply:
I thought it was safer, in cases where `parameters` might be a vector. I'll add the dependency to R 3.5.0. This will also cover cases where `WARNING: Added dependency on R >= 3.5.0 because serialized objects in
     serialize/load version 3 cannot be read in older versions of R.` appear in the check because of using `.rda` data for the vignette.

Reply:
Resolved in 6ec8ad7fce350054bb13fe75453bf3924b1fdbd4